cmd/vc-rest/startcmd/start.go

```diff
@@ -458,6 +481,8 @@ func buildEchoHandler(
 		return nil, fmt.Errorf("failed to instantiate new oidc4ci service: %w", err)
 	}
 
+	oidc4ciService = oidc4citracing.Wrap(oidc4ciService, conf.Tracer)
```

Comment:
I think this should be wrapped only if tracing is enabled, otherwise we're doing extra work (such as redacting) for no reason. (Same with other wrappers.)